cypress/plugins/index.js

```diff
@@ -17,10 +17,22 @@
 // }
 
 const clipboardy = require('clipboardy')
-module.exports = on => {
+const path = require('path')
+const fakeCameraPath = path.join(__dirname, '/../fixtures/face.mjpeg')
```

Comment:
Issue with relative path is resolved

How about my second comment - could we substitute fake video only for tests which passing through verification flow ? Or it could be just a global override ?

Reply:
only 1 test uses a verification flow, but if running a full run test, needed to set up a fake camera setting before starting the browser ('before:browser:launch'). since the browser starts once and all tests go in it in turn

Reply:
Okay